distribution/lib/Standard/Database/0.0.0-dev/src/Internal/Postgres/Postgres_Connection.enso

```diff
@@ -199,9 +215,32 @@ type Postgres_Connection
     jdbc_connection self = self.connection.jdbc_connection
 
     ## PRIVATE
-    drop_table : Text -> Nothing
-    drop_table self table_name =
-        self.connection.drop_table table_name
+       Drops a table.
+
+       Arguments:
```

Comment:
Does this need a "Dry Run" doc section?

Reply:
This is an private method and it does not have any 'dry run' semantics - it will throw `Forbidden_Operation` if executed in disabled Output context.

Reply:
No as it will hard fail if not enabled!